PlayerUI/Views/PUIPlayerView.swift

```diff
@@ -390,8 +387,8 @@ public final class PUIPlayerView: NSView {
     }
     
     deinit {
-        if let player = player {
-            teardown(player: player)
+        player.map {
```

Comment:
(Code style disagreement) I don't think using `map` here has any benefit over using `if let`, it's just a different style.